packages/castmill/lib/castmill_web/endpoint.ex

```diff
@@ -83,6 +83,7 @@ defmodule CastmillWeb.Endpoint do
   # TODO: we must cache the allowed origins at least for a few minutes or this
   # will be a huge performance bottleneck. The cache must use ETS to be shared.
   def getAllowedOrigins() do
-    Castmill.Networks.list_network_domains()
+    ["https://localhost" | Castmill.Networks.list_network_domains()]
```

Comment:
Is this debug code? I see an https protocol which is difficult to replicate locally, but in any case list_network_domains() should return the allowed domains correctly.

Reply:
It is needed to make it work on android where the local files are served from https://localhost